spec/requests/learning_units_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe LearningUnitsController, type: :request do
+  describe 'GET /show' do
+    let(:user) { create(:user) }
+    let(:learning_unit) { create(:learning_unit) }
+
+    before do
+      sign_in user
+    end
+
+    context 'when accesing to the learning unit page'
```

Comment:
It works fine, but just for convention, I'd use 
context 'blah blah context' **do**
  test_block
**end**
as I've seen it everywhere done like this

Reply:
I've corrected the style